config.py

```diff
@@ -21,6 +21,8 @@ def SQLALCHEMY_DATABASE_URI(self):
     SQLALCHEMY_RECORD_QUERIES = True
 
     SECRET_KEY = env_conf("SECRET_KEY", cast=str, default="12345")
+    # by default, access tokens do not expire
+    JWT_ACCESS_TOKEN_EXPIRES = env_conf('JWT_ACCESS_TOKEN_EXPIRES', cast=int, default=False) 
```

Comment:
cast=bool?

Reply:
It seems like it, but it's normally an integer value that corresponds to how long the token should be valid. False is just an override for "never expire."